### PR TITLE
Add test for grace login limit

### PR DESCRIPTION
**Pull request by b3lix**
ipatests: Add test for grace login limit 

Test user and pwpolicy entity for grace login limit setting.

Related: https://pagure.io/freeipa/issue/9211

Signed-off-by: Erik Belko <ebelko@redhat.com>

### CODE REVIEW

File: ipatests/test_webui/test_user.py [hunk 1]
```diff
@@ -446,6 +447,33 @@ def fill_password_dialog(self, password, current=None):
...
+    def test_grace_login_limit(self):
+        self.init_app()
+        self.add_record(group.ENTITY, [group.DATA])
+        # add record DATA8 already with passwordgracelimit
+        self.add_record(pwpolicy.ENTITY, [pwpolicy.DATA8])
+
+        self.navigate_to_record(group.PKEY)
+        # fill with values from DATA8, passwordgracelimit has value 42
+        self.fill_fields(pwpolicy.DATA8['mod'])
+
```
Comment (miskopo) [inline ipatests/test_webui/test_user.py:460]: add
```
        try:
            # click save if needed
            self.button_click('save')
        except AssertionError:
            # autosave active
            pass
```
 after modifying `pwpolicy`

**Review by miskopo — CHANGES_REQUESTED** [review]
Please see the inline comments

File: ipatests/test_webui/test_user.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
 import ipatests.test_webui.data_hbac as hbac
 import ipatests.test_webui.test_rbac as rbac
 import ipatests.test_webui.data_sudo as sudo
+import ipatests.test_webui.data_pwpolicy as pwpolicy
```
Comment (miskopo) [inline ipatests/test_webui/test_user.py:33]: just a nitpick, but there was a space before and also, the imports are in alphabetical order

**Review by miskopo — CHANGES_REQUESTED** [review]

**Comment by miskopo** [comment]
Great success! 

Please, provide links to the temp commits as a comment, remove the temp commit and I'll `ACK` when the CI finishes... **If** it finishes.

**Comment by b3lix** [comment]
[temp_commit_pwpolicy](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/a6a8ca92-5ea6-11ed-a070-fa163e86d7ce/)
[temp_commit_user](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/953b7d36-5ea6-11ed-b17b-fa163ed0f469)

**Comment by miskopo** [comment]
One last thing - in the commit message, you point to this very PR, however, the link should point to https://pagure.io/freeipa/issue/9211

**Review by miskopo — CHANGES_REQUESTED** [review]
Please see the comment above

**Comment by miskopo** [comment]
All is fine, now, seriously one last thing - please rebase this branch on top of upstream master to avoid potential conflicts

**Comment by miskopo** [comment]
master:

* d6a643b798deb673fa94bf8278fb291b42314562 ipatests: Add test for grace login limit

**Pull request closed without merging**